### DISCUSSION TITLE
Welcome to textlint Discussions!

### DISCUSSION THREAD

**Discussion by azu** [textlint/textlint#752] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dogeared** [comment]
Hello! My name is Micah. I am looking to create a specialized parser based on markdown. We have a documentation system that takes in markdown and spits out html. BUT, we have a few custom tags in the mix. Currently, we have no linter, so I am looking to bring some automation and error checking into the mix. Here's an example:


```
Use the code snippet below and plug in your API key. The API key is available on your **Organization Settings** page, on the **APIs** tab. The API key is of type `sdk`. For more information, see [Understanding API Keys](https://help.example.com/hc/en-us/articles/API-keys).

<div class="tabs">
{{# tab label="Java" }}
` ` `java
import com.example.FactoryBuilder;
import com.example.Client;
import com.example.ClientConfig;
import com.example.Factory;
 
ClientConfig config = ClientConfig.builder()
   .setBlockUntilReadyTimeout(10000) 
   .build();
 
Factory factory = FactoryBuilder.build("YOUR_API_KEY", config);
Client client = factory.client();
try {
    client.blockUntilReady();
} catch (TimeoutException | InterruptedException e) {
    // log & handle 
}
` ` `
{{/ end }}
</div>

Now you can start asking the SDK to evaluate treatments for your customers.

# Using the SDK
 
## Basic use

Use an if-else-if block as shown below and insert the code for the different treatments that you defined in the UI. Remember the final else branch in your code to handle the client returning the [control treatment](https://example.com/hc/en-us/articles/Control-treatment).
```

(Above I have spaces between backticks. In real life, the spaces wouldn't be there)

Ideally, everything between <div> and </div> (inclusive) would be part of an HTML block. I really just need to match the div tags.  Then, I can make sure that the {{#tab}} and {{/end}} match.

Right now, as soon as it hits the newline inside the HTML, it switches to paragraph:

![image](https://user-images.githubusercontent.com/364562/126262152-f7441859-1d2e-463f-8ebb-69e9e28eba0c.png)

Any help is much appreciated! Thanks.
